### DISCUSSION TITLE
Why hasn't it been updated for so long?

### DISCUSSION THREAD

**Discussion by zwl1619** [blitz-js/blitz#2724] [General] [answered]
Why hasn't it been updated for so long?

**Comment by prisis** [comment]
What do you mean?

**Comment by zwl1619** [comment, marked as answer]
The last update date is 10 days ago.

![image](https://user-images.githubusercontent.com/13177997/134149136-0100c9a1-3dc1-4dac-8023-ac8939c81544.png)

**Reply by flybayer** [reply]
Because I'm a man of many hats! 😁 Rest assured, just because there may be some gaps in time for commits to the canary branch, Blitz is very much maintained and working towards 1.0

**Comment by prisis** [comment]
What are your expectations, every hour or day a commit?

**Comment by flippyhead** [comment]
I guess it depends [where](https://github.com/blitz-js/blitz/pulls) you look ;)
